docs/dev/w3c-webdriver-capabilities.md

```diff
@@ -22,11 +22,14 @@ Some extended capabilities are not backwards-compatible with Selenium versions b
 * A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
 * Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
 
+
+## W3C WebDriver Protocol Compliance
+
 To ensure W3C WebDriver compliance:
 
 * Use Selenium version 3.11 or higher.
-* Switch from using JWP (legacy) to the newer W3C WebDriver Protocol. Mixing JWP with W3C will result in an [error](/dev/w3c-webdriver-capabilities/#common-errors).
-* Learn the name changes effective with the W3C protocol capabilities. For example, W3C uses `platformName`, while JWP uses `platform`. W3C uses `browserVersion`, while JWP uses `version`. We recommend reviewing our [Test Configuration Options](/dev/test-configuration-options) and the [official W3C Recommendations website](https://www.w3.org/TR/webdriver1/#capabilities).
+* Switch from using legacy JWP to the newer W3C WebDriver Protocol. Mixing JWP with W3C will result in an [error](/dev/w3c-webdriver-capabilities/#common-errors).
```

Comment:
```suggestion
* Switch from using legacy JSON Wire Protocol (JWP) to the newer W3C WebDriver Protocol. Mixing JWP with W3C will result in an [error](/dev/w3c-webdriver-capabilities/#common-errors).
```

Reply:
Hi @walkerlj0 , JWP is defined earlier in the doc (around line 13) so we wouldn't need to define it here again